eiffel-syntax-and-usage/the-links-object.md

```diff
@@ -75,7 +75,7 @@ __Description:__ Identifies the composition from which an artifact declared by [
 
 ### ENVIRONMENT
 __Required in:__ None  
-__Optional in:__ [EiffelArtifactCreatedEvent](../eiffel-vocabulary/EiffelArtifactCreatedEvent.md), 
+__Optional in:__ [EiffelArtifactCreatedEvent](../eiffel-vocabulary/EiffelArtifactCreatedEvent.md), [EiffelTestSuiteStartedEvent](../eiffel-vocabulary/EiffelTestSuiteStartedEvent.md), 
```

Comment:
Do we really want this? What does it mean, conceptually, that a test suite is executed in a particular environment? What happens if the test cases also reference an environment? What if they reference another one?